crmprtd/insert.py

```diff
@@ -129,7 +141,7 @@ def split(tuple_):
         return (tuple_[:i], tuple_[i:])
 
 
-def bisect_insert_strategy(sesh, obs):
+def bisect_insert_strategy(sesh, observations):
```

Comment:
Beware of changing variable names, lest you forever be known (i.e. `git blame`d) as the author of this code.

Reply:
That does mean you are forever stuck with bad things like confusing variable names.